lunar/release-stretch-build.yaml

```diff
@@ -12,6 +12,8 @@ notifications:
   - mikael+buildfarm@osrfoundation.org
   maintainers: true
 package_blacklist:
+package_whitelist:
+  - catkin
```

Comment:
My bad, I mimicked the existimg ones. This should be no indent instead of 2 spaces, is that right?